### PR TITLE
feat: [publicca] new module for publicca

### PR DESCRIPTION
**Pull request by suztomo**
- feat: initial generation of publicca
- build: add the publicca module to monorepo

### CODE REVIEW

File: java-publicca/.OwlBot.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+deep-remove-regex:
+- "/grpc-google-.*/src"
+- "/proto-google-.*/src"
+- "/google-.*/src"
```
Comment (lqiu96) [inline java-publicca/.OwlBot.yaml:19]: Shouldn't this prefixed with `/java-publicca`?

Comment (suztomo) [reply java-publicca/.OwlBot.yaml:19]: Good catch.

File: java-publicca/.OwlBot.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# limitations under the License.
+
+
+deep-remove-regex:
+- "/java-publicca/grpc-google-.*/src"
+- "/java-publicca/proto-google-.*/src"
+- "/java-publicca/google-.*/src"
+
+deep-preserve-regex:
+- "/java-publicca/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
```
Comment (suztomo) [inline java-publicca/.OwlBot.yaml:22]: @lqiu96 Fixed. Now they have "/java-publicca" at the beginning.

File: release-please-config.json [hunk 3]
```diff
@@ -384,6 +384,10 @@
       "component": "google-cloud-profiler",
       "skip-github-release": true
     },
+    "java-publicca": {
+      "component": "google-cloud-publicca",
+      "skip-github-release": true
```
Comment (meltsufin) [inline release-please-config.json:389]: What does `skip-github-release` do?

**Review by meltsufin — APPROVED** [review]

File: release-please-config.json (hunk 3 above)
Comment (suztomo) [reply release-please-config.json:389]: Not to create a GitHub release entry for now during the shadow mode.

**Pull request merged**